services/sources/publisher_backlist_service.py

```diff
@@ -54,13 +97,20 @@ def get_records_json(self,
         if offset == None:
             limit = 100
         
-        filter_by_formula = self.build_filter_by_formula_parameter(full_import, start_timestamp)
+        filter_by_formula = self.build_filter_by_formula_parameter(deleted=False, full_import=None, start_timestamp=None)
                 
         array_json_records = self.get_records_array(limit, filter_by_formula)
         
         return array_json_records
         
-    def build_filter_by_formula_parameter(self, full_import: bool=False, start_timestamp: datetime=None) -> str:
+    def build_filter_by_formula_parameter(self, deleted=None, full_import: bool=False, start_timestamp: datetime=None) -> str:
+        if deleted == True:
+            deleted_filter = f"IF(%7BDRB_Deleted%7D%20%3D%20TRUE(),%20TRUE(),%20FALSE())"
+            filter_by_formula = f"&filterByFormula={deleted_filter}"
+            return filter_by_formula
```

Comment:
You could just do `return "&filterByFormula=IF(%7BDRB_Deleted%7D%20%3D%20TRUE(),%20TRUE(),%20FALSE())"`

does this mean that it will only return records that are marked as deleted?

Reply:
I like the clarity of specifying the type and purpose of the filter that I'm adding to the formula. Also, it will only return records that are marked as deleted with a check mark which is equivalent to "true" in the JSON response from the Airtable API.